Userland/Libraries/LibWeb/Streams/WritableStreamDefaultWriter.cpp

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2023, Matthew Olsson <mattco@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibJS/Runtime/PromiseCapability.h>
+#include <LibWeb/Bindings/Intrinsics.h>
+#include <LibWeb/Bindings/WritableStreamDefaultWriterPrototype.h>
+#include <LibWeb/Streams/AbstractOperations.h>
+#include <LibWeb/Streams/WritableStream.h>
+#include <LibWeb/Streams/WritableStreamDefaultWriter.h>
+#include <LibWeb/WebIDL/ExceptionOr.h>
+
+namespace Web::Streams {
+
+WebIDL::ExceptionOr<JS::NonnullGCPtr<WritableStreamDefaultWriter>> WritableStreamDefaultWriter::construct_impl(JS::Realm& realm, JS::NonnullGCPtr<WritableStream> stream)
+{
+    auto writer = MUST_OR_THROW_OOM(realm.heap().allocate<WritableStreamDefaultWriter>(realm, realm));
+
+    // 1. Perform ? SetUpWritableStreamDefaultWriter(this, stream).
+    TRY(set_up_writable_stream_default_writer(*writer, stream));
+
+    return writer;
+}
+
+// https://streams.spec.whatwg.org/#default-writer-closed
+JS::GCPtr<JS::Object> WritableStreamDefaultWriter::closed()
```

Comment:
`const`

Reply:
(although "Arguably these should either return GCPtr<T const>, or not be const" also applies here, I suppose...)